### PR TITLE
Fixed import errors and updated to support python 3

### LINKED ISSUE #121: Missing import files and no backward python compatibility
# Expected Behavior

For python to deploy the backend properly, when running 'python main.py' without errors or crashes

# Current Behavior

1. Queue is not imported. This is because in python3, Queue is lowercase. Hence importing queue as queue fixes this.
import

2. print doesn't have parenthesis around it. Causing a crash in python3.
print

## Steps to Reproduce

1. Clone DroneSym
2. Navigate to DroneSym/dronesym-python/flask-api/src/
3. Open terminal here
4. Execute 'python main.py'

## Fix

https://github.com/scorelab/DroneSym/pull/120

## Context

Follow the Readme till 'python main.py' for same contextual setup as mine

Find this task on my website: http://itsrohan.com/gci/gcitasks

### PR DESCRIPTION
**Pull request by angeryrohan**
Fixes #121 

- Imported queue as queue to avoid import error

import

- Enabled backward compatible flask server. Did this by adding parenthesis around every print command in dronepool.py

### CODE REVIEW

**Comment by angeryrohan** [comment]
Earlier:

ree

Now:

ree 2

**Comment by rohancl** [comment]
@OshanIvantha   What do you think of the changes ?

**Comment by angeryrohan** [comment]
## End to End test for feature - [LogIn Authorization]<br> + Login Vulnerability Test
In the recent end to end testing of DroneSym, I wrote tests for ensuring URL protection by redirections and making sure that the login system works with both, user's and admin's credentials. Logged in and out several times, in different ways to ensure  security, the functioning of Login's UI and proper authorization.

## A look


ezgif com-gif-maker 1

**Comment by angeryrohan** [comment]
@OshanIvantha Please Review,
Thanks ^.^

**Review by swatisingh45 — COMMENTED** [review]
Squash your commits, please.

**Comment by angeryrohan** [comment]
@swatisingh45 Sure, I'm on it...

**Comment by angeryrohan** [comment]
@swatisingh45 Done.

**Comment by angeryrohan** [comment]
Recently, I made the page responsive at every scale, 
The web app now works flawlessly irrespective of the screen's dimensions.
EARLIER: 

screenshot from 2018-12-15 03-57-08

NOW:

webp net-gifmaker

**Comment by angeryrohan** [comment]
@OshanIvantha Please Review ^.^

File: dronesym-frontend/src/app/login/login.component.html [hunk 1]
```diff
@@ -1,15 +1,17 @@
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
 <div class="row login-background">
 	<div class="col s4 offset-s4 login-card">
 		<div class="card">
 			<div class="card-image">
-				<div class="logo-container">
-					<img src="./assets/img/dronesym_logo.png"/>
+				<div class="logo-container"><!-- -->
```
Comment (agentmilindu) [inline dronesym-frontend/src/app/login/login.component.html]: Do we need these `<!-- -->`?

File: dronesym-node/index.js [hunk 2]
```diff
@@ -57,7 +59,7 @@ app.use(function(err, req, res, next) {
 
 
 http.listen(3000, function(){
```
Comment (agentmilindu) [inline dronesym-node/index.js]: I think you have to use your new `port` variable here also if you really want it to be used

**Review by agentmilindu — CHANGES_REQUESTED** [review]

**Comment by agentmilindu** [comment]
I see lot of files related to `WiredTiger`, do we need them?

File: dronesym-node/index.js (hunk 2 above)
Comment (angeryrohan) [reply dronesym-node/index.js]: No we don't need these, This was just to test something.... I'll remove them right away ^.^

**Comment by angeryrohan** [comment]
Sorry, that's a terrible mistake there! I'll fix it right now....

**Comment by angeryrohan** [comment]
@agentmilindu , I have made the required changes..... 
And where exactly did you see the ```WiredTiger``` files, Please give me the location

**Comment by agentmilindu** [comment]
@rohancl  See the higlighted places. Plus, I assuemm `.wt` files are also related to WildTiger.
 
image

**Comment by angeryrohan** [comment]
@agentmilindu , Seems that that was an auto-generated file by MongoDB, Anyway I've deleted them... Have a look now ^.^

**Review by agentmilindu — APPROVED** [review]
LGTM

**Comment by agentmilindu** [comment]
@OshanIvantha Can you check and merge this?

**Review by malithsen — APPROVED** [review]

**Comment by angeryrohan** [comment]
@agentmilindu @OshanIvantha @malithsen  All required code quality tests are now passing ^.^ ....

**Review by ivantha — CHANGES_REQUESTED** [review]
@rohancl This looks good to go. 
However can you squash the commits please? 
Also there is an unnecessary package-lock.json file created at the root.

**Comment by angeryrohan** [comment]
Oh, thanks for pointing that out...
I removed the package-lock.JSON file and squashed the commits
Thanks, ^.^
@OshanIvantha

**Review by ivantha — APPROVED** [review]

**Pull request merged**